lib/order.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'restaurant'
+
+class Order
+
+  attr_reader :total
+
+  def initialize(restaurant_klass: Restaurant)
+    @total = 0
+    @restaurant_klass = restaurant_klass
+    @restaurant = @restaurant_klass.new
+  end
+
+  def see_menu
+    @restaurant.dishes
+  end
+
+  def place_order(item, quantity)
+    item.each{ |x|
+      if see_menu.include?(x)
```

Comment:
Style/Next: Use `next` to skip iteration. (https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)